inc/branding/namespace.php

```diff
@@ -76,6 +77,38 @@ function add_color_scheme() {
 	);
 }
 
+/**
+ * Remove all unused default WP admin color schemes for a user.
+ *
+ * If a user has previously chosen one of the WP default admin color schemes,
+ * leave that scheme as is. Remove all other WP default admin color schemes,
+ * so that they can't be selected.
+ */
+function remove_wp_admin_color_schemes() {
+	global $_wp_admin_css_colors;
+
+	// Use color slugs as keys for a quick access.
+	$admin_color_schemes = [
+		'fresh' => true,
```

Comment:
It looks like the value `true` is completely unused here so you don't need an associative array.